include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (433)
+#define OPENTHREAD_API_VERSION (434)
```

Comment:
Can you rebase and increment again? Thanks!

Reply:
@jwhui Done. Thanks for review.